### PR TITLE
Improving the Developer Experience

### PR DESCRIPTION
**Pull request by mewdriller**

### CODE REVIEW

**Comment by csteinert** [comment]
:ship:

File: netjets-user-group/slides.md [hunk 1]
```diff
@@ -54,29 +54,316 @@ Presented by:
...
+## Developer Experience (DX)
+
+Your productivity is a precious commodity; use it wisely.
+
+1. Focus on the task at hand
+2. Use purpose-built tools
+3. Reduce cognitive overhead
+
+Note: 
+- We talk a lot about UX, but let's talk about DX.
```
Comment (bradpearson) [inline netjets-user-group/slides.md]: Just so we're clear - are you officially coining 'DX' as a thing? Like 'fetch'?

**Comment by bradpearson** [comment]
+1

File: netjets-user-group/slides.md (hunk 1 above)
Comment (mewdriller) [inline netjets-user-group/slides.md]: To be clear, I am not officially coining it. I'm just parroting back everything [Dan Abramov](https://github.com/gaearon) says. :wink:

**Pull request merged**